dbt_af/operators/macros.py

```diff
@@ -10,10 +10,13 @@
 
 
 class DbtRunMacroOperation(DbtIntervalActionOperator):
-    def __init__(self, **kwargs):
+    retries = 2
```

Comment:
Where is it used?

Reply:
removed static attributes from operations